src/main/java/org/dcsa/bkg/model/transferobjects/BookingConfirmationTO.java

```diff
@@ -31,5 +34,5 @@
 
   private List<ChargeTO> charges;
 
-  private List<CarrierClauseTO> carrierClauses;
+  private List<CarrierClause> carrierClauses;
```

Comment:
we are now mixing TO objects and data objects in the TO response. Would be better if a TO object is made from other TO objects and not include data objects

Reply:
Good point, I'll do that.